docs/guides/notebooks/configuration/groups.rst

```diff
@@ -0,0 +1,129 @@
+############################
+User groups for file sharing
+############################
+
+Create and manage closed user groups in order to share private files in the ``/project`` directory.
+
+.. _user-group-create:
+
+Create a closed group
+=====================
+
+Comanage webpage
+----------------
+
+* In a browser, navigate to `id.lsst.cloud <https://id.lsst.cloud>`_ and log in.
+
+* In the left menu sidebar, click on "Groups" and then "My Groups".
+
+* At right, a list of all the groups associated with your account will appear.
+
+* In the top horizontal menu, click on "Add Group".
+
+* Set the group properties:
+
+  * The group name should be short and must start with ``g_``.
+
+  * Write a short description.
+
+  * Leave the status as the default "Active".
+
+  * Leave the checkbox next to "Open" unselected, to create a closed group.
+
+  * Leave the checkbox next to "Require All for Nested Memberships" unchecked; this can be changed later if nested sub-groups are created.
+
+* Click the blue "Add" button.
+
+
+The next time you enter the Notebook Aspect, this group will be accessible.
+
+
+Command line
+------------
+
+At this time, it is not possible to create new groups from the terminal command line in the Notebook Aspect.
```

Comment:
I would take that out the "at this time" - this will never be a feature since we consume group information from CoManage. 

It also feels a bit weird to me to tell people they can't do something (the list of things they can't do is... infinite) but up to you,

Reply:
Ah ok good to know. I actually wasn't sure if it wasn't possible from a terminal at all, or I just couldn't figure out how. That's sort of why the sub-sections for command-line existed... but now I've removed them as sections. The page still mentions that using the terminal for groups is not possible (because users might have done group management this way in other systems and would expect it).